jaeger_client/tracer.py

```diff
@@ -217,7 +217,7 @@ def start_span(self,
                                parent_id=parent_id, flags=flags,
                                baggage=baggage)
         span = Span(context=span_ctx, tracer=self,
-                    operation_name=operation_name,
+                    operation_name=operation_name,  # type:ignore
```

Comment:
why ignore?

Reply:
So the operation_name can be None at that point, but the Span constructor doesn't specify it as optional. I guess I need to make it optional... What do you think?

Reply:
```suggestion
                    operation_name=operation_name or '',
```